Chisel/CHLDuplicateDetection.m

```diff
@@ -0,0 +1,47 @@
+//
+//  CHLDuplicateDetection.m
+//  Chisel
+//
+//  Created by Greg Pstrucha on 3/1/16.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+#if __has_feature(objc_arc)
+#error CHLDuplicateDetection.m expects ARC to be disabled.
```

Comment:
Ah you are right, in the previous version I used MRR intentionally but end-solution definitely does not need it. I'll clean it up